source/slang/slang-options.cpp

```diff
@@ -289,6 +289,10 @@ void initCommandOptions(CommandOptions& options)
         { OptionKind::Language,     "-lang", "-lang <language>", "Set the language for the following input files."},
         { OptionKind::MatrixLayoutColumn, "-matrix-layout-column-major", nullptr, "Set the default matrix layout to column-major."},
         { OptionKind::MatrixLayoutRow,"-matrix-layout-row-major", nullptr, "Set the default matrix layout to row-major."},
+        { OptionKind::ZeroInitialize, "-zero-initialize", nullptr, 
+        "Set all variables to zero."
+        "Structs will set all struct-fields without an init expression to 0."
+        "All variables will call their default constructor if not explicitly initialized as usual."},
```

Comment:
```
Set all variables to zero
```
is not the correct description for the implementation, is it?
It will be better with "Initialize all variables to zero".